docs/source/routing/performance/caching/distributed.mdx

```diff
@@ -138,15 +145,25 @@ supergraph:
         pool_size: 4 # Optional, defaults to 1
 ```
 
-#### Timeout
+### Timeout
 
 Connecting and sending commands to Redis are subject to a timeout, set by default to 500ms, that can be overriden.
 
 
-#### TTL
+### TTL
 
 The `ttl` option defines the default global expiration for Redis entries. For APQ caching, the default is no expiration, while for query plan caching, the default expiration is set to 30 days.
 
+When enabling distributed caching, consider how frequently you publish new schemas and configure the TTL accordingly. When new schemas are published, the router [pre-warms](/router/configuration/in-memory-caching#cache-warm-up) the in-memory and distributed caches but **does not** invalidate existing cached query plans in the distributed cache,
```

Comment:
Use contractions for `not` instead of adding emphasis

```suggestion
When enabling distributed caching, consider how frequently you publish new schemas and configure the TTL accordingly. When new schemas are published, the router [pre-warms](/router/configuration/in-memory-caching#cache-warm-up) the in-memory and distributed caches but doesn't invalidate existing cached query plans in the distributed cache,
```